cluster/centos/config-build.sh

```diff
@@ -29,7 +29,7 @@ FLANNEL_VERSION=${FLANNEL_VERSION:-"0.6.1"}
 ETCD_VERSION=${ETCD_VERSION:-"3.0.9"}
 
 # Define k8s version to use.
-K8S_VERSION=${K8S_VERSION:-"1.3.7"}
+K8S_VERSION=$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)
```

Comment:
I think it's intended to be a static version number so that with a standard invocation of the script, it will definitely deploy a functional cluster. If a user wants to use this script to deploy a newer version of Kubernetes, they can set the `K8S_VERSION` env var before running the script (e.g.: `K8S_VERSION=1.8.1 ./cluster/centos/config-build.sh`).

Reply:
If that is the case, should we update the static number to something that is currently in the support window? 1.6.11 or 1.7.8 would make sense.

Reply:
That sounds sensible to me. Is this area of the codebase covered by e2e tests?

Reply:
Also the flannel/etcd/Docker versions need to be supported by the Kubernetes version, so making the latter dynamic will likely cause problems in the feature.